CMakeLists.txt

```diff
@@ -209,11 +213,15 @@ elseif(UNIX)
 endif()
 
 if(USE_LAPACK)
-  add_definitions(-DMXNET_USE_LAPACK=1)
-  list(APPEND mxnet_LINKER_LIBS lapack)
+  if(BLAS STREQUAL "Open" OR BLAS STREQUAL "open")
+    add_definitions(-DMXNET_USE_LAPACK=1)
+  elseif()
+    find_package(LAPACK REQUIRED)
```

Comment:
Does LAPACK require CBLAS (atlas) for CPU?

Reply:
ATLAS should be included a few LAPACK operations.But I don't know it is enough.